agent/connectionuptime/service_test.go

```diff
@@ -267,3 +267,68 @@ func compareFloatWithTolerance(a, b float32) bool {
 
 	return diff < tolerance
 }
+
+func TestCalculationConnectionUpTimeWhenCleanupMethodIsNotCalled(t *testing.T) {
+	now := time.Now()
+	tests := []struct {
+		name             string
+		setOfConnections map[time.Time]bool
+		expectedUpTime   float32
+		windowPeriod     time.Duration
+		toTime           time.Time
+	}{
+		{
+			name: "should return 50% uptime when window period is 10s, and 100% uptime when window period is 5s",
```

Comment:
It does not seem we're testing the second part of the test "and 100% uptime when window period is 5s".
The same applies to the following test.

Shouldn't the uptime be 80% with 5s window period?

Reply:
my bad, it's copy-paste issues

renamed test